app/src/main/res/values-de/strings.xml

```diff
@@ -67,7 +67,9 @@
     <string name="helper_type_button_transcript">"Einen Anruf übersetzen!"</string>
     <string name="helper_call_overview_screen_title">"Offene Anrufe:"</string>
     <string name="seeker_phone_call_text">"Rufen Sie bitte <br><b>%1$s</b></br>um Ihre Einkaufsliste aufzuzeigen."</string>
-    <string name="seeker_type_screen_title">"Wie würden Sie gerne Ihre Liste erstellen?"</string>
+    <string name="seeker_type_screen_title">"Wie würden Sie gerne
```

Comment:
was this intended?

Reply:
yes, if you compare the screen designs you will see there are some intentional line breaks. The text in some of the buttons already uses intentional line breaks

Reply:
To be honest i don't think that it was an intentional line break. But even then usage of \n would be better in my opinion. We are using POEditor for localization management and i'm not sure if it can handle such a line break. What do you think?

Reply:
I agree on using an actual newline string literal. The last change on line 73/74 for `Make a phone call` was from `schroepf` on 4/15, and since he works on IOS I assumed it would come from changing strings in POEditor. I'll revert the line breaks on the strings I changed